common/src/interlocked_hl.c

```diff
@@ -91,6 +91,7 @@ IMPLEMENT_MOCKABLE_FUNCTION(, INTERLOCKED_HL_RESULT, InterlockedHL_WaitForValue,
     else
     {
         int32_t current_value;
+        bool continue_eval = true;
```

Comment:
>bool continue_eval = true; [](http://example.com/codeflow?start=8&length=26)

I highly suspect that the new code can be reformulated without the use of "continue_eval". Probably a combination of "continue" and "break" keywords would do it.

Reply:
taken care of